packages/design-system/src/styles/_layout.scss

```diff
@@ -1,4 +1,6 @@
 $grid-columns: 12 !default;
```

Comment:
This is out of scope of this ticket, but I think it might be a good idea to generate a doc note at the top of these files to explain that it was generated from the tokens package.

Reply:
I started working on porting CMS.gov and am now more accutely aware of the stuff that is just magic in our code

Reply:
This would be easy to add, i'll work it in here.